lib/operations/mongo_client_ops.js

```diff
@@ -439,10 +439,14 @@ function createTopology(mongoClient, topologyType, options, callback) {
     }
 
     const MongoClient = loadClient();
-    const connectionString =
-      os.platform() === 'win32'
-        ? 'mongodb://localhost:27020/?serverSelectionTimeoutMS=1000'
-        : 'mongodb://%2Ftmp%2Fmongocryptd.sock/?serverSelectionTimeoutMS=1000';
+    let connectionString;
+    if (mongoClient.s.options.autoEncryption.cryptdConnectionString != null) {
```

Comment:
maybe we can replace `mongoClient.s.options.autoEncryption.cryptdConnectionString` with `options.autoEncryption.cryptdConnectionString`?

Reply:
replaced.